### PR TITLE
fix aptos fund command

### PR DESCRIPTION
**Pull request by 0xbe1**
### Test Plan
No test needed

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3509)

### CODE REVIEW

File: developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md [hunk 1]
```diff
@@ -207,14 +207,14 @@ Aptos is now set up for account 18B61497FD290B02BB0751F44381CADA1657C2B3AA6194A0
 You can fund an account with the faucet via the CLI with either an account or a profile:
 
 ```bash
-$ aptos account fund --account B9BD2CFA58CA29BCE1D7ADD25FCE5C62220604CD0236FE3F90D9DE91ED9FB8CB
+$ aptos account fund-with-faucet --account B9BD2CFA58CA29BCE1D7ADD25FCE5C62220604CD0236FE3F90D9DE91ED9FB8CB
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md:210]: I am not sure if this is accurate. See here: https://github.com/aptos-labs/aptos-core/blob/84dbcacf8172b62ffbd430de7390d5c5a0aea15e/crates/aptos/src/test/mod.rs#L6

**Review by rajkaramchedu — CHANGES_REQUESTED** [review]

File: developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md (hunk 1 above)
Comment (rajkaramchedu) [reply developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md:210]: Did the `fund` option not work for you? And did you try and confirm that `fund-with-faucet` option worked for you? I would be surprised if it did.

Comment (0xbe1) [reply developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md:210]: `fund-with-faucet` works for me, while `fund` doesn't.

i am using aptos-account 0.3.1 - is that the right version?
```
$ aptos account --help
aptos-account 0.3.1
Tool for interacting with accounts

This tool is used to create accounts, get information about the account's resources, and transfer
resources between accounts.

USAGE:
    aptos account <SUBCOMMAND>

OPTIONS:
    -h, --help
            Print help information

    -V, --version
            Print version information

SUBCOMMANDS:
    create
            Command to create a new account on-chain
    create-resource-account
            Command to create a resource account on-chain
    fund-with-faucet
            Command to fund an account with tokens from a faucet
    help
            Print this message or the help of the given subcommand(s)
    list
            Command to list resources, modules, or other items owned by an address
    transfer
            Command to transfer coins between accounts
```

Comment (rajkaramchedu) [reply developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md:210]: You are correct. I had the older version. I confirmed `fund-with-faucet` with the 0.3.1 version. Thanks!

**Review by rajkaramchedu — APPROVED** [review]

File: developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md (hunk 1 above)
Comment (gregnazario) [reply developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md:210]: Yes this was changed, and the docs not updated.  Thanks for catching

**Review by gregnazario — APPROVED** [review]

**Pull request merged**